aviary/subsystems/aerodynamics/aerodynamics_builder.py

```diff
@@ -339,14 +340,106 @@ def get_parameters(self, aviary_inputs=None, phase_info=None):
                         params[Aircraft.Design.DRAG_POLAR] = drag_opts
 
             if method == 'computed':
-                param_vars = [Aircraft.Nacelle.CHARACTERISTIC_LENGTH,
-                              Aircraft.Nacelle.FINENESS,
-                              Aircraft.Nacelle.LAMINAR_FLOW_LOWER,
-                              Aircraft.Nacelle.LAMINAR_FLOW_UPPER,
-                              Aircraft.Nacelle.WETTED_AREA]
-                for var in param_vars:
+
+                # Parameters for drag computation.
+                core_inputs = [
```

Comment:
would it make sense to define these lists further up/out, so they can be imported by other files?